cdap-standalone/pom.xml

```diff
@@ -241,7 +253,7 @@
                   <overWriteSnapshots>false</overWriteSnapshots>
                   <overWriteIfNewer>true</overWriteIfNewer>
                   <excludeGroupIds>org.apache.hbase,asm</excludeGroupIds>
-                  <excludeArtifactIds>zkclient,servlet-api,cdap-cli,cdap-explore-jdbc</excludeArtifactIds>
+                  <excludeArtifactIds>zkclient,servlet-api,cdap-cli,cdap-explore-jdbc,cdap-kafka</excludeArtifactIds>
```

Comment:
Make the dependency on `cdap-kafka` in provided scope, then you don't need to exclude in here.